src/clj/game/cards/ice.clj

```diff
@@ -1339,6 +1339,14 @@
                                 (when (<= 3 (+ (:advance-counter card 0) (:extra-advance-counter card 0)))
                                   (end-run state side)))}]}
 
+   "Masvingo"
+   {:implementation "Number of subs is manual"
+    :advanceable :always
+    :abilities [{:label "Gain subroutines"
+                 :msg (msg "gain " (:advance-counter card 0) " subroutines")}]
+    :events {:rez {:effect (effect (add-prop card :advance-counter 1))}}
```

Comment:
This triggers on every rez of any card 😛 . This should probably just be in the `:effect` of the card itself, which triggers when the card is rezzed.